src/backend/__tests__/utils.test.ts

```diff
@@ -198,7 +198,7 @@ describe('backend/utils.ts', () => {
         getBytesFromMB(totalSizeMB),
         lastProgressTime
       )
-      expect(etaString).toEqual('00:00:04')
+      expect(etaString).toEqual(expect.stringMatching(/00:00:0[4,5]/))
```

Comment:
maybe we should make this more general?
```js
expect(etaString).toEqual(expect.stringMatching(/00:00:0\d/))
```

Reply:
I haven't seen it more than 1 second off. Theoretically if there's no delay, it should be 05, but with the delay it's 04. If we did `0\d` then it could handle being up to 5 seconds off, but I think if we have >1 second off, it's more than likely a problem with the function we're testing

I'd recommend just matching 04 and 05 and if we have another failure at 03 where it is a timing issue and not a bug with calculateEta, then we change it to match 03-05

Reply:
Roger that! Thanks for the explanation 😄